moto/route53domains/validators.py

```diff
@@ -0,0 +1,1044 @@
+import re
+from datetime import datetime, timedelta, timezone
+from ipaddress import ip_address
+from typing import Dict, List
+
+from moto.core.common_models import BaseModel
+from moto.moto_api._internal import MotoRandom
```

Comment:
```suggestion
from moto.moto_api._internal import mock_random as random
```
That allows you to use the regular methods further down (`random.uuid4()`), makes it a bit easier to read.